### PR TITLE
Overload load to accept zip files

### PR DESCRIPTION
**Pull request by FynnBe**

### CODE REVIEW

**Review by constantinpape — COMMENTED** [review]
So with this `load_node` / `load_raw_node` excepts both the path to the rdf or the zipped folder?
Can you please also add a test that checks that it actually works?

File: bioimageio/spec/model/v0_1/raw_nodes.py [hunk 1]
```diff
@@ -19,6 +19,9 @@
 FormatVersion = Literal["0.1.0"]
 
 
+Dependencies = Dependencies
```
Comment (constantinpape) [inline bioimageio/spec/model/v0_1/raw_nodes.py:22]: ???

File: bioimageio/spec/model/v0_3/raw_nodes.py [hunk 2]
```diff
@@ -28,6 +28,7 @@
 # same as general RDF
 Badge = Badge
 CiteEntry = CiteEntry
+Dependencies = Dependencies
```
Comment (constantinpape) [inline bioimageio/spec/model/v0_3/raw_nodes.py:31]: Ah, I guess this is a common pattern :D. But it looks weird enough that a comment is warranted.

**Review by constantinpape — APPROVED** [review]
Some minor comments, looks good overall.

File: bioimageio/spec/model/v0_3/raw_nodes.py (hunk 2 above)
Comment (FynnBe) [reply bioimageio/spec/model/v0_3/raw_nodes.py:31]: yeah... I don't know what the way to go is there...  just importing it and not using it looks like an unrequired import.

Comment (constantinpape) [reply bioimageio/spec/model/v0_3/raw_nodes.py:31]: yeah, maybe just write a comment to explain it

Comment (FynnBe) [reply bioimageio/spec/model/v0_3/raw_nodes.py:31]: added comments in 44f4d5e

**Pull request merged**